WebRoot/css/zm.css

```diff
@@ -1637,6 +1669,9 @@ INPUT[type="text"].search_input-expanded:focus {
 	overflow:auto;
 }
 
+.attBubbleHolder > span:not(:first-child) {
+	@AttachmentChipSpacing@
```

Comment:
We can have have alternative work around for this.
But have couple of questions ? And open for discussion, here .
Are you talking about 'external' skin developer/ contributor i.e not from Zimbra organization ?